client/my-sites/checkout/checkout-thank-you/domains/domain-mapping-thank-you.jsx

```diff
@@ -53,12 +59,12 @@ const DomainMappingThankYou = ( { domain, translate } ) => {
 							),
 							stepCta: (
 								<Button
-									href={ '#' }
+									href={ emailManagement( selectedSite.slug, domain.name ) }
```

Comment:
Can you use `emailManagementPurchaseNewEmailAccount` here instead? We split out the intentional "purchase new email" content into the new path.